rolling/release-build.yaml

```diff
@@ -16,7 +16,7 @@ notifications:
   - ros2-buildfarm-rolling@googlegroups.com
   maintainers: true
 sync:
-  package_count: 262
+  packages: [ros_desktop]
```

Comment:
The checked package name gets [converted into a platform package name](https://github.com/ros-infrastructure/ros_buildfarm/blob/833969e9f9a4e907059c4377472ae0d018f2c884/scripts/release/check_sync_criteria.py#L79-L81) before checking the repository index so I expect that the package.xml name is expected in this list as with other package names in the build files.

It will _always_ trip me up that our default variants are `ros_base`, `ros_core`, and `desktop`. :unamused: